### ISSUE TITLE
Kernel Dead while training a model!

### ISSUE THREAD

**Issue by mingjun1120** [amaiya/ktrain#388]
I am using GPU to run my program which I have installed `tensorflow-gpu==2.5.0`, `tensorflow==2.5.0` and `ktrain==0.27.2` in my virtual environment (tensorflowGPU) through Anaconda Prompt but then the Kernel dead when comes to training a model. I have installed the correct CUDA toolkit and CUDAnn in my Windows system but end up kernel dead😢

How to solve this issue?

**Issues:**
![image](https://user-images.githubusercontent.com/54136320/130398987-1d2c1476-b486-42ad-a5d5-25e83abbedfd.png)

**Virtual Environment Installation**
![image](https://user-images.githubusercontent.com/54136320/130399574-96122d91-9e21-4d07-9f1d-e931d53227d7.png)

**Comment by amaiya** [comment]
Hello:  Are there any errors in the terminal from where you're running jupyter notebook?   When you run your code in the terminal (outside of jupyter notebook), what errors are displayed?

**Comment by mingjun1120** [comment]
> Hello: Are there any errors in the terminal from where you're running jupyter notebook? When you run your code in the terminal (outside of jupyter notebook), what errors are displayed?

I did try to run the code in Google Colab and everything works fine but when I try to run in local machine using GPU instead of CPU then this problem happened.

**Comment by mingjun1120** [comment]
> Hello: Are there any errors in the terminal from where you're running jupyter notebook? When you run your code in the terminal (outside of jupyter notebook), what errors are displayed?

For now, I haven't tried to run the program inside the terminal. If I get you right, are u saying to write those codes inside Anaconda Prompt by activating the python editing via `python`?

**Comment by amaiya** [comment]
> For now, I haven't tried to run the program inside the terminal. If I get you right, are u saying to write those codes inside Anaconda Prompt by activating the python editing via python?

Yes.  Or, put the code in a `.py` file and run it to see what errors are displayed: `python mycode.py`.
